### ISSUE TITLE
train_contrast_acc during pre-training

### ISSUE THREAD

**Issue by tarunn2799** [google-research/simclr#74]
What does train_contrast_acc during the pre-training stage mean? Since there are no labels to calculate the ground truth accuracy or anything.

**Comment by chentingpc** [comment]
it is the accuracy for predicting the positive augmented example pair (conditioned on one of the two examples).
